crates/subspace-gateway/src/main.rs

```diff
@@ -17,11 +18,18 @@ async fn main() -> anyhow::Result<()> {
     init_logger();
     raise_fd_limit();
 
+    info!("Subspace Gateway");
+    info!("✌️  version {}", env!("CARGO_PKG_VERSION"));
+    info!("❤️  by {}", env!("CARGO_PKG_AUTHORS"));
+
     let command = Command::parse();
 
     match command {
-        Command::Run(run_options) => {
-            commands::run::run(run_options).await?;
+        Command::Rpc(run_options) => {
+            commands::rpc::run(run_options).await?;
+        }
+        Command::Http(run_options) => {
+            commands::http::run(run_options).await?;
```

Comment:
Are we sure that we won’t want both RPC and HTTP served by the same process?

Reply:
I’m guessing this is also a question for @clostao 🙂

Reply:
It's okey to have them separate. The consumers of each services are different: one is the object-mapping-indexer and the other the file retriever (the service responsible of building a file from object mappings content) so there is no need to have them being in the same process for this case.